.github/ISSUE_TEMPLATE/1_bug_report.yml

```diff
@@ -0,0 +1,37 @@
+name: Bug report
+description: Create a report to help us improve
+labels: ["bug"]
+body:
+  - type: textarea
+    attributes:
+      label: Summary
+      description: |
+        A clear and concise description of the bug, including a minimal reproducible example.
+
+        Please include your full output with the complete error message.
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Platform
+      description: What operating system and architecture are you using? (see `uname -orsm`)
```

Comment:
Stating the obvious, but `uname` won't work on Windows. Perhaps a python sys.platform equivalent

Reply:
This is also borrowed from `uv`. It will tell you about wsl, which is nice e.g. `Linux 5.15.167.4-microsoft-standard-WSL2 x86_64 GNU/Linux`

`sys.platform` doesn't seem to give much information, at least for me (just `darwin`), so I'm not sure it's much more useful than people hopefully just telling us Windows version x?